test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/DefaultEditorTemplatesTest.cs

```diff
@@ -112,6 +199,82 @@ public void ObjectTemplateEditsSimplePropertiesOnObjectByDefault()
             Assert.Equal(expected, HtmlContentUtilities.HtmlContentToString(result));
         }
 
+        // Expect almost the same HTML as in ObjectTemplateEditsSimplePropertiesOnObjectByDefault(). Only difference is
+        // the <div class="editor-label">...</div> is not present for Property1.
+        [Fact]
+        public void ObjectTemplateSkipsLabel_IfDisplayNameIsEmpty()
```

Comment:
This was a test gap. Would have worked (almost by accident) before the #5317 fix.